dash-pipeline/bmv2/dash_acl.p4

```diff
@@ -29,6 +30,8 @@ match_kind {
     table table_name { \
         key = { \
             meta. ## table_name ##_dash_acl_group_id : exact @name("meta.dash_acl_group_id:dash_acl_group_id"); \
+            meta.dst_tag_map : ternary @name("meta.dst_tag_map:dst_tag"); \
```

Comment:
match type: optional

Reply:
Why is here optional?
1. We would like a mask operation.
E.G. If the source IP of packet belong to tag 1 and 2, and this ACL rule is applied to tag 2 and 3, it should be matched. If another ACL rule is applied to tag 3 and 4, it should be mismatched.
2. If the tag was provided, the target rule could match all tags.
I'm not sure whether the optional match_kind can meet above behaviors.

Reply:
A case is if a prefix belong is to a tag, A, B, C, D (1111) and an ACL rule is applied to tag A, C(1010). 
If my understanding is correct, to the match_kind, optional. the mask should be 1111, which implies `tag (1111) & mask(1111) != target_tag(1010)`.
But to the match_kind, ternary, the mask will be 1010. So the rule will be matched due to `tag(1111) & mask(1010) = target_tag(1010)`.